Gemfile.lock

```diff
@@ -7,41 +7,73 @@ PATH
 GEM
   remote: https://rubygems.org/
   specs:
-    diff-lcs (1.1.3)
-    ffi (1.1.5)
-    guard (1.4.0)
-      listen (>= 0.4.2)
-      thor (>= 0.14.6)
-    guard-rspec (2.1.0)
-      guard (>= 1.1)
-      rspec (~> 2.11)
-    listen (0.5.3)
-    mock_redis (0.5.4)
-    rake (0.9.2.2)
-    rb-fsevent (0.9.2)
-    rb-inotify (0.8.8)
-      ffi (>= 0.5.0)
-    redis (3.2.0)
-    rspec (2.11.0)
-      rspec-core (~> 2.11.0)
-      rspec-expectations (~> 2.11.0)
-      rspec-mocks (~> 2.11.0)
-    rspec-core (2.11.1)
-    rspec-expectations (2.11.3)
-      diff-lcs (~> 1.1.3)
-    rspec-mocks (2.11.3)
-    thor (0.16.0)
-    timecop (0.5.2)
+    coderay (1.1.3)
+    diff-lcs (1.5.0)
+    ffi (1.15.5)
+    formatador (1.1.0)
+    guard (2.18.0)
+      formatador (>= 0.2.4)
+      listen (>= 2.7, < 4.0)
+      lumberjack (>= 1.0.12, < 2.0)
+      nenv (~> 0.1)
+      notiffany (~> 0.0)
+      pry (>= 0.13.0)
+      shellany (~> 0.0)
+      thor (>= 0.18.1)
+    guard-compat (1.2.1)
+    guard-rspec (4.7.3)
+      guard (~> 2.1)
+      guard-compat (~> 1.1)
+      rspec (>= 2.99.0, < 4.0)
+    listen (3.7.1)
+      rb-fsevent (~> 0.10, >= 0.10.3)
+      rb-inotify (~> 0.9, >= 0.9.10)
+    lumberjack (1.2.8)
+    method_source (1.0.0)
+    mock_redis (0.31.0)
+      ruby2_keywords
+    nenv (0.3.0)
+    notiffany (0.1.3)
+      nenv (~> 0.1)
+      shellany (~> 0.0)
+    pry (0.14.1)
+      coderay (~> 1.1)
+      method_source (~> 1.0)
+    rake (13.0.6)
+    rb-fsevent (0.11.1)
+    rb-inotify (0.10.1)
+      ffi (~> 1.0)
+    redis (4.6.0)
+    rspec (3.11.0)
+      rspec-core (~> 3.11.0)
+      rspec-expectations (~> 3.11.0)
+      rspec-mocks (~> 3.11.0)
+    rspec-core (3.11.0)
+      rspec-support (~> 3.11.0)
+    rspec-expectations (3.11.0)
+      diff-lcs (>= 1.2.0, < 2.0)
+      rspec-support (~> 3.11.0)
+    rspec-mocks (3.11.1)
+      diff-lcs (>= 1.2.0, < 2.0)
+      rspec-support (~> 3.11.0)
+    rspec-support (3.11.0)
+    ruby2_keywords (0.0.5)
+    shellany (0.0.1)
+    thor (1.2.1)
```

Comment:
### [thor](http://whatisthor.com/)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 0.16.0 → 1.2.1

[_[change-log](https://github.com/rails/thor/releases/tag/v1.2.1), [source-code](https://github.com/rails/thor/tree/v1.2.1), [gem-diff](https://my.diffend.io/gems/thor/0.16.0/1.2.1)_]





<details>
<summary>Commits</summary>

A change of **712** commits. See the full changes on [the compare page](https://github.com/rails/thor/compare/v0.16.0...v1.2.1).

These are the first 10 commits:
- (1e1d85d) [Fix incorrect use of Process::exit. This fixes open issue #2…](https://github.com/rails/thor/commit/1e1d85dbfd5ecb56e8a4bfc29767614284924794)
- (2644b85) [lock hashdiff to <0.3.6 to fix travis](https://github.com/rails/thor/commit/2644b853e5870ba8d27e8cbc5fbdfb6ae9a236c2)
- (cb3bb7d) [update ruby version in travis config](https://github.com/rails/thor/commit/cb3bb7dbd968c0ab8b65b1059e77e58e0fb890fa)
- (5c4388e) [Merge pull request #583 from lostapathy/fix_travis](https://github.com/rails/thor/commit/5c4388ecc7ebd8bd13c716233228542a94aed058)
- (61d4866) [Merge pull request #584 from lostapathy/bump_travis_versions](https://github.com/rails/thor/commit/61d486636dcab88a575192527421969adfa734a1)
- (16a075e) [Fixed misspelling words.](https://github.com/rails/thor/commit/16a075eec394318f32aa5b944dc72f4a29f060ec)
- (0e2615f) [Merge pull request #586 from hsbt/fix-misspell](https://github.com/rails/thor/commit/0e2615fca90ea277c09f2e9bf5c4eba156253d8a)
- (713dbb6) [Use correct line numbers for `class_eval` and `module_eval` …](https://github.com/rails/thor/commit/713dbb6f1fad4ea8e892642e02d8bc1b8a60e909)
- (70021d0) [Deprecate safe_level of ERB.new in Ruby 2.6](https://github.com/rails/thor/commit/70021d00e1f906b0252a73c90e3b09711c794fbf)
- (47d7440) [Address #595 by duplicating string objects](https://github.com/rails/thor/commit/47d7440c433387a91bf857545679ef5f964002b6)


</details>